client/hosting-overview/components/quick-actions-card.tsx

```diff
@@ -72,6 +83,11 @@ const QuickActionsCard: FC = () => {
 					href={ statsUrl }
 					text={ translate( 'See Jetpack Stats' ) }
 				/>
+				<Action
+					icon={ <SidebarCustomIcon icon="dashicons-wordpress-alt hosting-overview__dashicon" /> }
+					href={ site ? site.options?.admin_url || `${ site.URL }/wp-admin` : '' }
```

Comment:
We can use [getSiteAdminUrl](https://github.com/Automattic/wp-calypso/blob/e0816b9dc65192ec4a848d1c097a46428bf0d416/client/state/sites/selectors/get-site-admin-url.ts) 😄

Reply:
Updated!